src/content/structured/components/text-input/code.mdx

```diff
@@ -68,7 +68,7 @@ export const snippetsIconValueMaxLength = [
   {
     language: "Web component",
     snippet: `<ic-text-field 
-    max-length="25" 
+    maxLength={25} 
```

Comment:
To keep consistent with our other web-component examples, we usually use kebab-case for attributes which are more than one word. Also HTML attributes take string so it's previous state was fine.

Reply:
Thank you for informing me on this I will ensure there's consistency with future changes and make these minor changes in this PR.